lib/retriable/exponential_backoff.rb

```diff
@@ -25,19 +25,16 @@ def initialize(opts = {})
 
     def intervals
       intervals = Array.new(tries) do |iteration|
-        [base_interval * multiplier**iteration, max_interval].min
+        [base_interval * multiplier ** iteration, max_interval].min
       end
 
-      return intervals if rand_factor.zero?
```

Comment:
Yeah, it makes sense in `randomize`, since it should exit early if rand_factor is 0, but I also think it makes sense in `intervals` too. If it can skip the map, `randomize` method invocation, allocating a new array, why not have it be more efficient?